core/amber/src/main/scala/edu/uci/ics/texera/web/resource/WorkflowWebsocketResource.scala

```diff
@@ -172,25 +193,30 @@ class WorkflowWebsocketResource {
       context
     )
 
-//    texeraWorkflowCompiler.init()
+    //    texeraWorkflowCompiler.init()
     val violations = texeraWorkflowCompiler.validate
     if (violations.nonEmpty) {
       send(session, WorkflowErrorEvent(violations))
       return
     }
 
+    clearCache(session)
```

Comment:
I think it is not good to clear cache inside execute workflow. try to move it to another place.

Reply:
This cache is related to the workflow execution results so it should be cleared when the same workflow is executed again. Could you suggest a place to put the code?